netty/src/main/java/io/grpc/netty/GrpcHttp2HeadersUtils.java

```diff
@@ -353,44 +358,42 @@ public Http2Headers add(CharSequence csName, CharSequence csValue) {
     @Override
     public CharSequence get(CharSequence csName) {
       AsciiString name = requireAsciiString(csName);
-      checkArgument(!isPseudoHeader(name), "Use direct accessor methods for pseudo headers.");
+      if (isPseudoHeader(name)) {
+        return getPseudoHeader(name);
+      }
       if (equals(TE_HEADER, name)) {
         return te;
       }
       return get(name);
     }
 
-    private void addPseudoHeader(CharSequence csName, CharSequence csValue) {
-      AsciiString name = requireAsciiString(csName);
-      AsciiString value = requireAsciiString(csValue);
+    private AsciiString getPseudoHeader(AsciiString name) {
```

Comment:
Should we annotate this with `@Nullable`?

Reply:
This class doesn't use `@Nullable`. In my mind it'd be misleading to add it to this one method. The get() method just above this one and probably many more would need to be checked.